src/server/dev.ts

```diff
@@ -29,10 +30,10 @@ const nbsFile = args.play
 
 const app = express()
 const server = http.createServer(app)
-const sioNetwork = sio(server)
+const sioNetwork = sio(server, { parser: NUClearProxyParser } as any)
 
 // Initialize socket.io namespace immediately to catch reconnections.
-WebSocketProxyNUClearNetServer.of(WebSocketServer.of(sioNetwork.of('/nuclearnet')), {
```

Comment:
What happened to `/nuclearnet`?

Reply:
I don't see the need to have a namespace once we start using our own custom parser. It's just more bytes on the wire and no other socket.io could connect anyway without using this parser.

Reply:
I introduced a namespace so that nuclearnet communication would be separate from nusight communication. See the WIP recording PR: https://github.com/NUbots/NUsight2/pull/90/files#diff-7c0807743476996169d50ee8dd9c0addR71

The roadmap for recording/playback is pretty unclear atm though.